### ISSUE TITLE
Several issues in k0s status sub-command

### ISSUE THREAD

**Issue by kke** [k0sproject/k0s#970]
* There's some problem in the version detection, giving `/proc/xxxx/exe: Permission denied` error in some cases.
* The output format is serialized into the output:
    ```yaml
    # k0s status -o yaml
    version: v1.21.1+k0s.0
    pid: 12348
    ppid: 1
    role: controller+worker
    sysinit: linux-systemd
    stubfile: ""
    output: yaml
    ```
* The term "stubfile" [seems incorrect](https://en.wikipedia.org/wiki/Stub_file).
* There's a `String()` function that does not return a string.
* The default "unformatted" output is pretty close to the YAML output, could just output yaml instead.

**Comment by jasmingacic** [comment]
Some of this has been addressed in #985

**Comment by jasmingacic** [comment]
@kke can you confirm this has been fixed  with #985

**Comment by kke** [comment]
Yeah this can be closed, the status thing has changed competely.
